README.md

```diff
@@ -289,10 +295,34 @@ __Arguments__
 
 ---------------------------------------
 
+<a name="notFound"/>
+#### Redbird##notFound(callback)
+
+ Gives Redbird a callback function with two parameters, the HTTP request 
+ and response objects, respectively, which will be called when a proxy route is
+ not found. The default is
+```javascript
+    function(req, res){
+      res.statusCode = 404;
+      res.write('Not Found');
+      res.end();
+    };
+```
+.
+
+__Arguments__
+
+```javascript
+    src {Function(req, res)} The callback which will be called with the HTTP
```

Comment:
I kind of made up this formatting for documenting a parameter that's a callback. Not sure if there's a standard for this or anything.